modules/thunder_demo/thunder_demo.install

```diff
@@ -5,19 +5,6 @@
  * Contains.
  */
 
-/**
- * Install thunder demo content.
- */
-function thunder_demo_install() {
```

Comment:
sure that you no longer need this? if you no longer have home/403/404 pages fine, but note that the new default content does not have fixed ID's, I moved our version of this logic to a \Drupal\default_content\Event\DefaultContentEvents::IMPORT event, and then act on content with a specific uuid being installed.

Reply:
Yes, we are removing these pages and use the default.

Thx for the hint about the events, that might be useful in the future.